Dockerfile

```diff
@@ -1,3 +1,6 @@
+# ARGs used in FROM must be declared before the first FROM
+ARG CLOUDPOSSE_PACKAGES_VERSION="0.86.0"
```

Comment:
The reason we did not do this is b/c it then thwarts dependabot. We can increase the frequency of dependabot checks, which would be better.